### PR TITLE
Example game provider

### PR DESCRIPTION
**Pull request by gustavste**
- Example provider with basic components for creating a provider

### CODE REVIEW

File: example/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# When creating a new provider module
+
+Start with creating a folder under the provider-folder with the provider name.
+
+In [this folder](./example-game-provider/) you will find `example-game-provider` with some basic building blocks for creating a new provider module.
+- `router.go` - set up provider wallet api as well as endpoints operator will call, such as gamelaunch
+- `controller.go` - request parsing and validation from wallet calls
+- `wallet_service.go` - mainly a mapping service converting the provider specific api to work with the Valkyrie PAM api
+- `provider_service.go` - in the other way the Valkyrie operator api needs to be converted to work with the provider specifics, such as gamelaunch
+- `config.go` - define auth and provider specific from the configuration.
```
Comment (gnirb) [inline example/README.md:10]: Something about this sentence is a bit off, maybe a word missing?

File: example/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# When creating a new provider module
+
+Start with creating a folder under the provider-folder with the provider name.
+
+In [this folder](./example-game-provider/) you will find `example-game-provider` with some basic building blocks for creating a new provider module.
```
Comment (gnirb) [inline example/README.md:5]: Maybe mention models.go too, since that seem to be pretty standard for us

File: example/README.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+- `router.go` - set up provider wallet api as well as endpoints operator will call, such as gamelaunch
+- `controller.go` - request parsing and validation from wallet calls
+- `wallet_service.go` - mainly a mapping service converting the provider specific api to work with the Valkyrie PAM api
+- `provider_service.go` - in the other way the Valkyrie operator api needs to be converted to work with the provider specifics, such as gamelaunch
+- `config.go` - define auth and provider specific from the configuration.
+
+Another step that needs to be done is defining what configuration is needed for this specific provider.
```
Comment (gnirb) [inline example/README.md:12]: Maybe link to a working example here just to get some more context?

File: example/README.md (hunk 1 above)
Comment (gustavste) [reply example/README.md:10]: Referring to the keys in config. Will make it more clear

**Review by gnirb — APPROVED** [review]
Great stuff!

**Review by goodnightanderson — APPROVED** [review]
Good stuff! 👍

**Pull request merged**